stylesheets/site-structure.scss

```diff
@@ -90,7 +90,7 @@ h1.panel-title {
 }
 
 .screenshot > a > img {
-    max-height: 480px;
+    max-height: 640px;
```

Comment:
No, the problem wasn't that the image was "to small" _per se_; the issue was that this rule prevented images from being full-width on the page. That made it look extremely awkward when full-width text was before and after an image that wasn't full-width.

The project page that originally made me notice the issue isn't using the screenshot template anymore, so even if other pages are affected I'm, not going to bother with fixing it at this point.